installation/resources/README.md

```diff
@@ -0,0 +1,21 @@
+# Installer resources
+
+## Overview
+
+This directory contains resources used by installer.
+
+## Installer CRs
+
+Files starting with `installer-cr-*` contain different component configurations that can be provided to installer. Currently available configurations:
+
+| Configuration | Description | Officially supported |
+|----------------|------|------|
+| `installer-cr.yaml.tpl` | Complete local Kyma installation | ✅ |
+| `installer-cr-cluster.yaml.tpl` | Complete cluster Kyma installation | ✅ |
```

Comment:
Maybe a bullet point list would be better, with compass jobs marked as `(Experimental)`